packages/@aws-cdk/custom-resource-handlers/test/custom-resources/aws-custom-resource/aws-sdk-v3-handler.test.ts

```diff
@@ -709,3 +711,37 @@ test('invalid v2 service name throws explicit error', async () => {
 
   expect(request.isDone()).toBeTruthy();
 });
+
+test('automatic Uint8Array conversion when necessary', async () => {
+  const kmsMock = mockClient(KMSClient);
+  kmsMock.on(EncryptCommand).resolves({
+    CiphertextBlob: new TextEncoder().encode('dummy-data'),
+    KeyId: 'key-id',
+    EncryptionAlgorithm: 'SYMMETRIC_DEFAULT',
+  });
+
+  await handler({
+    ...eventCommon,
+    RequestType: 'Create',
+    ResourceProperties: {
+      ServiceToken: 'token',
+      Create: JSON.stringify({
+        service: 'KMS',
+        action: 'encrypt',
+        parameters: {
+          KeyId: 'key-id',
+          Plaintext: 'dummy-data',
+        },
+      } satisfies AwsSdkCall),
+    },
+  }, {} as AWSLambda.Context);
+
+  expect(kmsMock).toHaveReceivedCommandWith(EncryptCommand, {
+    KeyId: 'key-id',
+    Plaintext: new Uint8Array([
```

Comment:
wait the field named `Plaintext` is Uint8Array? 😕

Reply:
Yes!